### PR TITLE
PR: Add `GitHub CI` and `@dependabot` issue #1

### PR DESCRIPTION
**Pull request by nelsonic**
@LuchoTurtle this quick update just fixes the broken Build/Badge noted in https://github.com/dwyl/deltamin/issues/1#issuecomment-1646867793
image

Please sense-check and merge when you can ahead of work on #10 🙏

### CODE REVIEW

**Comment by nelsonic** [comment]
Eventually got the build/CI to pass by removing _all_ the superfluous. ✂️ 
Hopefully _some_ of the groundwork is done for #10 🤞

**Comment by nelsonic** [comment]
@LuchoTurtle this should be mergeable now. 🤞 (thanks! 🙏)

**Comment by nelsonic** [comment]
@LuchoTurtle merge this one first before you embark on porting to `Dart` #10 🙏

File: .github/workflows/ci.yml
Comment (LuchoTurtle) [inline .github/workflows/ci.yml:1]: Can you just change the `ci.yml` to `js.yml` so it's just a tad bit more discernable ?

**Review by LuchoTurtle — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml
Comment (nelsonic) [reply .github/workflows/ci.yml:1]: Sure. Will do that now. But you can also do it when you uncomment the `.github/workflows/dart.yml` for #10 💭

Comment (nelsonic) [reply .github/workflows/ci.yml:1]: Done. ✅

**Comment by nelsonic** [comment]
PR comment addressed. Please review + merge when you can. 🙏

File: README.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-# `deltamin`
+# `deltamin` Δ
 
 Minify your Delta format data
 before sending it down the wire
 or saving it to a datastore.
 
-[![Build Status](https://img.shields.io/travis/com/dwyl/deltamin/master.svg?style=flat-square)](https://travis-ci.com/dwyl/deltamin)
-[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/deltamin/master.svg?style=flat-square)](http://codecov.io/github/dwyl/deltamin?branch=master)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/dwyl/terminate/js.yml?label=build&style=flat-square&branch=main)
```
Comment (LuchoTurtle) [inline README.md]: Mayhaps you meant `GitHub Workflow Status`?

This label is using `dwyl/terminate`'s repo handle.

**Review by LuchoTurtle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nelsonic) [reply README.md]: Good catch. Copy-paste fail. Fixing.

Comment (nelsonic) [reply README.md]: I was just testing your attention to detail. 😜 
Fixed.

**Review by LuchoTurtle — APPROVED** [review]
lgtm!

**Pull request merged**